endpointmanager/pkg/chplquerier/vendorquerier_integration_test.go

```diff
@@ -198,6 +202,9 @@ func Test_GetCHPLVendors(t *testing.T) {
 	// test context ended
 	// also checks what happens when an http request fails
 
+	hook := logtest.NewGlobal()
+	expectedErr := "Got error:\nmaking the GET request to the CHPL server failed: Get \"https://chpl.healthit.gov/rest/developers?api_key=tmp_api_key\": context canceled\n\nfrom URL: https://chpl.healthit.gov/rest/developers?api_key=tmp_api_key"
+
 	tc, err = basicVendorTestClient()
 	th.Assert(t, err == nil, err)
 	defer tc.Close()
```

Comment:
Maybe just to keep things consistent, move the lines with variables hook and expectedErr to under where the test client is made, since this is the ordering in the other files containing this test